uniswap-v2-core/contracts/UniswapV2Pair.sol

```diff
@@ -35,6 +47,27 @@ contract UniswapV2Pair is IUniswapV2Pair, UniswapV2ERC20 {
         unlocked = 1;
     }
 
+    modifier onlyFactory() {
+        require(msg.sender == factory, 'Vexchange: FORBIDDEN');
+        _;
+    }
+
+    function setRecoverer(address _recoverer) external onlyFactory {
```

Comment:
Should order functions the same as factory:
 1. Set swap
 2. Set platform
 3. Set recoverer

Reply:
Resolved